inst/include/interface/rcpp/rcpp_objects/rcpp_tmb_dnorm_likelihood.hpp

```diff
@@ -0,0 +1,220 @@
+/*
```

Comment:
This file needs more comments and documentation. It's very sparse and is not clear at all what it does.

Reply:
Added documentation